client/me/security-checkup/index.jsx

```diff
@@ -51,4 +48,10 @@ module.exports = React.createClass( {
 			</Main>
 		);
 	}
-} );
+}
+
+SecurityCheckup.displayName = 'SecurityCheckup';
+
+SecurityCheckup.mixins = [ observe( 'userSettings' ) ];
```

Comment:
I wasn't sure about setting mixins in this way so I wanted to take a peek at wether or not they can be set in this way...
After a little investigation I found these results by adding a console.log to the observe mixin...
**Before**:
<img width="1208" alt="before" src="https://cloud.githubusercontent.com/assets/4335450/19644872/df55b578-9a3d-11e6-9578-a8858fcb58d4.png">

**After**:
<img width="1210" alt="after" src="https://cloud.githubusercontent.com/assets/4335450/19644687/e54d6ff8-9a3c-11e6-9f77-e72cbd3ee9e8.png">

It appears that this mixin is not applied after the ES6 update :(
I'm not sure what the repercussions/regressions will be by not having the mixin assigned.

I'll be very keen to help out with getting this working - I'll come back with an update if I can crack it nicely :)